extensions/amp-list/0.1/amp-list.js

```diff
@@ -692,6 +708,29 @@ export class AmpList extends AMP.BaseElement {
     }
   }
 
+  /**
+   * @private
+   */
+  setLoadMoreReload_() {
+    if (this.loadMoreOverflow_) {
+      this.mutateElement(() => {
+        this.loadMoreOverflow_.classList.toggle('amp-visible', true);
+        listen(this.loadMoreOverflow_, 'click',
```

Comment:
Should this ever be unlistened?

Reply:
Yes, it should be unlistened after the load succeeds, but not if it fails. Updated the code to reflect this.